src/sample/Models/QuestionContainer.java

```diff
@@ -0,0 +1,8 @@
+package sample.Models;
+
+import java.util.HashMap;
+
+public class QuestionContainer {
```

Comment:
This can be a singleton so that it can be used a shared model across the application.